snippets/ertp/guide/test-issuers-and-mints.js

```diff
@@ -0,0 +1,219 @@
+import '@agoric/install-ses';
+import test from 'ava';
+
+// #region import
+import { makeIssuerKit, MathKind } from '@agoric/ertp';
+// #endregion import
+
+test('ertp guide issuers and mints makeIssuerKit', async t => {
+  // #region makeIssuerKit
+  const {
+    issuer: quatloosIssuer,
+    mint: quatloosMint,
+    amountMath: quatloosAmountMath,
+    brand: quatloosBrand,
+  } = makeIssuerKit('quatloos');
+  // This is merely an amount, describing assets.
+  // It does not create new assets.
+  const quatloos2 = quatloosAmountMath.make(2);
+  // Non-fungible asset, which needs an amountMath
+  // of kind 'MathKind.STRING_SET'
+  const {
+    mint: titleMint,
+    issuer: titleIssuer,
+    amountMath: titleAmountMath,
+  } = makeIssuerKit('alamedaCountyPropertyTitle', MathKind.STRING_SET);
+  // #endregion makeIssuerKit
+
+  t.truthy(quatloosIssuer);
+  t.truthy(quatloosMint);
+  t.truthy(quatloosBrand);
+  t.deepEqual(quatloos2, { brand: quatloosBrand, value: 2 });
+  t.truthy(titleMint);
+  t.truthy(titleIssuer);
+  t.truthy(titleAmountMath);
+});
+
+test('ertp guide issuers and mints getBrand', async t => {
+  // #region getBrand
+  const { issuer: quatloosIssuer, brand: quatloosBrand } = makeIssuerKit(
+    'quatloos',
+  );
+  // myQuatloosBrand === quatloosBrand
+  const myQuatloosBrand = quatloosIssuer.getBrand();
+  // #endregion getBrand
+
+  t.is(quatloosBrand, myQuatloosBrand);
+});
+
+test('ertp guide issuers and mints getAllegedName', async t => {
+  // #region getAllegedName
+  const { issuer: quatloosIssuer } = makeIssuerKit('quatloos');
+  const quatloosIssuerAllegedName = quatloosIssuer.getAllegedName();
+  // quatloosIssuerAllegedName === 'quatloos'
+  t.is(quatloosIssuerAllegedName, 'quatloos');
+  // #endregion getAllegedName
```

Comment:
Not clear the `t.is()` should be within the snippet. Many other places, there's a comment asserting something about the results without a test to validate it being in the snippet.

Reply:
I'll remove. I was considering including more test assertions in the code examples but there's no need to have a comment and an assertion